src/telliot_feeds/sources/manual_sources/string_query_manual_source.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+
+from telliot_feeds.datasource import DataSource
+from telliot_feeds.dtypes.datapoint import datetime_now_utc
+from telliot_feeds.dtypes.datapoint import OptionalDataPoint
+
+
+class StringQueryManualSource(DataSource[Optional[str]]):
+    async def fetch_new_datapoint(self) -> OptionalDataPoint[str]:
+        text = None
+        msg = "Type your string query response:"
+        print(f"{msg}\n")
+
+        while text is None:
+            usr_inpt = input()
+
+            # not sure what to catch here but possible bytes that could be passed via something other than the cli
+            # since everything will be interepreted to string when passed through the cli.
+            if len(usr_inpt) > 1 and isinstance(usr_inpt, str):
```

Comment:
Any string goes, even if they want to put `"b'blah'"` on chain. Maybe just a message to let them know they're going to report an empty string if that's the case. Anything else just preview the string they'll be putting on chain before